webrtc/interfaces.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html>
+  <head>
+    <title>"WebRTC IDL Tests</title>
```

Comment:
Many tests are more minimal:
https://github.com/w3c/web-platform-tests/blob/master/dom/historical.html
https://github.com/w3c/web-platform-tests/blob/master/fullscreen/api/document-onfullscreenerror.html
https://github.com/w3c/web-platform-tests/blob/master/webrtc/RTCPeerConnection-createDataChannel.html

But, you get to pick your style, just wanted to make sure you weren't including stuff because you though it's required.